zxlive/rules.py

```diff
@@ -122,7 +122,11 @@ def bialgebra(v_list):
     return g
 
 def Hadamard_slide(v):
-    
+    '''
+    g: BaseGraph[[VT,ET]]
+    v: node or vertex 
+    returns: absorbs the hadamard appropriately if the node phase is pi/2
```

Comment:
Same here, the function returns a new graph with the `Hadamard absorbed if node phase is pi/2`